nautobot/core/cli.py

```diff
@@ -132,6 +132,10 @@ def _configure_settings(config):
     if settings.METRICS_ENABLED and "postgres" in settings.DATABASES["default"]["ENGINE"]:
         settings.DATABASES["default"]["ENGINE"] = "django_prometheus.db.backends.postgresql"
 
+    # Create fake db for job logs. This uses the default db, but allows us to save logs within
+    # transaction.atomic().
+    settings.DATABASES["job_logs"] = settings.DATABASES["default"]
```

Comment:
This worries me in that it is a magic setting that we're trying to move away from.

Reply:
Well, if we move away from running jobs in Atomic Transactions then this can go away as well. This is not meant to be modified, so that is why we stuck it here.